tmail-backend/saas/saas-rabbitmq/src/test/java/com/linagora/tmail/saas/rabbitmq/subscription/SaaSDomainSubscriptionDeserializerTest.java

```diff
@@ -318,19 +319,6 @@ void parseMissingRequiredDomainShouldThrowException() {
                 .hasMessageContaining("Failed to parse SaaS subscription domain message");
         }
 
-        @Test
-        void parseMissingRequiredEnabledShouldThrowException() {
```

Comment:
This does not work anymore... In the current structure because in the subscription message all fields except `domain` became optional, if `enabled` is missing then it's considered a sub message and not a cancellation message anymore